src/integrationTest/java/io/aiven/kafka/connect/jdbc/oracle/VerifyDeleteIT.java

```diff
@@ -0,0 +1,317 @@
+ * limitations under the License.
+ */
+
+package io.aiven.kafka.connect.jdbc.oracle;
+
+import java.sql.SQLException;
+import java.time.Duration;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.Future;
+
+import org.apache.kafka.clients.consumer.ConsumerRecords;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.clients.producer.RecordMetadata;
+import org.apache.kafka.common.TopicPartition;
+
+import io.aiven.connect.jdbc.JdbcSinkConnector;
+
+import org.apache.avro.Schema;
+import org.apache.avro.generic.GenericData;
+import org.apache.avro.generic.GenericRecord;
+import org.assertj.db.type.Table;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+import static org.assertj.db.api.Assertions.assertThat;
+import static org.awaitility.Awaitility.await;
+import static org.junit.Assert.assertNotSame;
+
+public class VerifyDeleteIT extends AbstractOracleIT {
+
+    private static final String TEST_TOPIC_NAME = "SINK_TOPIC";
+    private static final String CONNECTOR_NAME = "test-sink-connector";
+    private static final int TEST_TOPIC_PARTITIONS = 1;
+
+    private static final Schema VALUE_RECORD_SCHEMA = new Schema.Parser().parse("{\n"
+            + "  \"type\": \"record\",\n"
+            + "  \"name\": \"record\",\n"
+            + "  \"fields\": [\n"
+            + "    {\n"
+            + "      \"name\": \"name\",\n"
+            + "      \"type\": \"string\"\n"
+            + "    },\n"
+            + "    {\n"
+            + "      \"name\": \"value\",\n"
+            + "      \"type\": \"string\"\n"
+            + "    }\n"
+            + "  ]\n"
+            + "}");
+
+    private static final String DROP_TABLE = String.format("DROP TABLE IF EXISTS %s", TEST_TOPIC_NAME);
+    private static final String CREATE_TABLE = String.format("CREATE TABLE \"%s\" (\n"
+            + "    \"id\" NUMBER NOT NULL,\n"
+            + "    \"name\" VARCHAR2(255) NOT NULL,\n"
+            + "    \"value\" VARCHAR2(255) NOT NULL,\n"
+            + "PRIMARY KEY(\"id\")"
+            + ")", TEST_TOPIC_NAME);
+
+    private Map<String, String> sinkConnectorConfigForDelete() {
+        final Map<String, String> config = basicConnectorConfig();
+        config.put("name", CONNECTOR_NAME);
+        config.put("connector.class", JdbcSinkConnector.class.getName());
+        config.put("topics", TEST_TOPIC_NAME);
+        config.put("pk.mode", "record_key");
+        config.put("pk.fields", "id"); // assigned name for the primitive key
+        config.put("delete.enabled", String.valueOf(true));
+        return config;
+    }
+
+    private void assertNotEmptyPoll(final Duration duration) {
+        final ConsumerRecords<?, ?> records = consumer.poll(duration);
+        assertNotSame(ConsumerRecords.empty(), records);
+    }
```

Comment:
What's the purpose of this method? Doesn't it just verify that some data has landed in Kafka, which we should already be reasonably sure of with our tracking of producer futures in `sendTestData` and similar methods?

Reply:
(And if this method is unnecessary, we can probably also remove all calls to `consumer::assign` from these tests.)